libmscore/clef.cpp

```diff
@@ -175,7 +175,7 @@ void Clef::layout()
                   break;
             case ClefType::TAB:                            // TAB clef
                   // on tablature, position clef at half the number of spaces * line distance
-                  yoff = lineDist * (lines - 1) * .5;
+                  yoff = lineDist * (lines - 1) * 0.77 ;
```

Comment:
@lasconic I had changed 0.5 to 0.77 to correct the position of TAB as shown in the image
![tabclef-position](https://cloud.githubusercontent.com/assets/14026308/21650633/9d95616c-d2cb-11e6-9fac-bb066c714b6a.png)